Kick.cpp

```diff
@@ -11,8 +11,10 @@ Kick::Kick( Kick const & src ) : ACommand(src)
 
 Kick::Kick(Server &server, Client &user, std::string rawInput) : ACommand(server, user, rawInput)
 {
+
 	std::cout << "KICK overloaded constructor" << std::endl;
 	std::cout << "client socketFD : " << this->_author->getSocketFD() << std::endl;
```

Comment:
Please encapsulate in
#if SHOW_CONSTRUCTOR
#endif 

(same for destructor)

Reply:
Thank you for your input